packages/dds/tree/src/feature-libraries/sequence-field/moveEffectTable.ts

```diff
@@ -558,10 +416,15 @@ export function splitMarkOnInput<TMark extends InputSpanningMark<unknown>>(
             fail(`Unable to split ${type} mark of length 1`);
         case "ReturnTo": {
             const newId = genId();
-            splitMoveSrc(moveEffects, mark.id, [
-                { id: mark.id, count: length },
-                { id: newId, count: remainder },
-            ]);
+            splitMove(
```

Comment:
Shouldn't we also record a move effect to split this Dest if `recordMoveEffect` is true?

Reply:
Fixed